config/example/config.txt

```diff
@@ -323,12 +323,6 @@ PLAYER_REROUTE_CAP 0
 ## Disable the loading of space ruins
 #DISABLE_SPACE_RUINS
 
-## Minimum number of space ruins levels to generate
```

Comment:
Both unused.

Reply:
https://github.com/ParadiseSS13/Paradise/blob/b16ec46bff0d1abcc4a92c44ee7887c6e45158f0/code/controllers/master_controller.dm#L51

Reply:
Ahh, I see the issue. The config lists them as "EXTRA_SPACE_RUIN**S**_LEVELS_MIN" whereas they're EXTRA_SPACE_RUIN_LEVELS_MIN in the code. Note the missing S.

Reply:
I wanna know how this hasn’t been Uber broken all these months what the hell

Reply:
This might explain the disturbing lack of space ruins in rounds :thinking:

Reply:
@Mitchs98 are you saying none appear or just a very small amount of them appear?

If you say none, you are not correct as I see them most rounds as mechanic, and their spawns are logged in DD

Reply:
I mean very small amount. I'm aware that _some_ appear :p.